Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+                        }, {
+                            "answer": "Objective-C",
+                            "vote_url": "/questions/1/choices/3",
+                            "votes": 0
+                        }, {
+                            "answer": "Ruby",
+                            "vote_url": "/questions/1/choices/4",
+                            "votes": 0
+                        }
+                    ]
                 }
+```
 
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
-
-    + Response 200
+The next resource in our group allows you to view the detail of a question directly with an ID of a question.
 
-        [Gists Collection][]
+```markdown
+## Question [/questions/{question_id}]
+```
 
-    ### Create a Gist [POST]
-    To create a new Gist simply provide a JSON hash of the *description* and *content* attributes for the new Gist.
+### URI Template
 
-    + Request (application/json)
-
-            {
-                "description": "Description of Gist",
-                "content": "String content"
-            }
-
-    + Response 201 (application/hal+json)
-
-        [Gist][]
-
-At this point you should be familiar with everything in this listing except the definition and discussion of the query parameter *"since"*:
-
-<a name="query_parameters"></a>
-#### Query Parameters
-
-    ## Gists Collection [/gists{?since}]
-
-As with the URI parameters the Query parameters are defined in the [URI Template][] format. The difference here is that a query parameter name is preceded by the *question mark* and it is always at the end of the URI Template.
-
-> **NOTE:** To define multiple query parameters simply comma-separate their names. For example `{?month,year}`.
-
-#### Action-specific Query Parameters
-
-    ### List All Gists [GET]
-    + Parameters
-        + since (optional, string) ... Timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ` Only Gists updated at or after this time are returned.
-
-Often a resource query parameter is specific to just one of the resource actions. In this case you can discuss it in the relevant action only. No matter where you discuss a query parameter it follows exactly the same syntax as with [URI Parameters][].
-
-## Subsequent Resource
-The last missing piece in the expected functionality is the Star Resource and its actions. Utilizing everything we have learned so far the Star Resource and its actions are defined as follows:
-
-    ## Star [/gists/{id}/star]
-    Star resource represents a Gist starred status.
-
-    The Star resource has the following attribute:
-
-    - starred
+The URI for “Question” resource uses a variable component, expressed by [URI Template][]. In this case, we have an id variable as `{question_id}`. Representing this resource is for `/questions/{question_id}` where `question_id` can be substituted for a real `id`.
```

Comment:
Doesn't even make sense to the Kyle of Thursday.

![Double Facepalm](http://i.stack.imgur.com/jiFfM.jpg)